src/views/ManageList.jsx

```diff
@@ -1,7 +1,69 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function ManageList({ listPath }) {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
+	const [message, setMessage] = useState('');
+
+	const handleSubmit = async (event) => {
+		event.preventDefault();
+		try {
+			await addItem(listPath, { itemName, daysUntilNextPurchase });
+			setMessage('Item was successfully saved to the database.');
+		} catch (error) {
+			setMessage('There was an error saving the item to the database.');
```

Comment:
Expanding on @shuveksha-tuladhar's comment about the set message: Currently if you create an item, the message displays. But then if you create a second item, and the message text doesn't change, it's not immediately obvious if your second item was submitted or not. Not a problem for now, but something to consider for down the line when y'all are revising your designs.